utils/raiseTelegramError.ts

```diff
@@ -1,13 +1,19 @@
-// import { showMessage } from 'react-native-flash-message';
+import { addAlert } from '../redux/alertsSlice';
+import { AppDispatch } from '../redux/store';
+import { Alert } from '../types';
 
 const customData = require('./telegramErrors.json');
 
-export const raiseTelegramError = (error_message: string) => {
+export const raiseTelegramError = (
+  error_message: string,
+  dispatch: AppDispatch,
+) => {
   const message = customData.descriptions[error_message]
     ? customData.descriptions[error_message]
     : 'Unknown error';
-  // showMessage({
-  //   message: message,
-  //   type: 'danger',
-  // });
+  const alertMessage: Alert = {
```

Comment:
Is it needed to provide a type?
```suggestion
  const alertMessage = {
```

Reply:
Yes, it is needed.
![image](https://github.com/inzynierka-home-iot/frontend-rncli/assets/49338133/0b143e5f-7279-4471-a78c-7cb2abe081b6)